tests/templates/output-success-llvm.json

```diff
@@ -0,0 +1,162 @@
+<kevm>
```

Comment:
why is this file here? it shouldn't actually be using krun when running the llvm backend, right?

Reply:
The `./kevm interpret ` command does `k-bin-to-text` on the output if it sees a non-zero exit code, and then the `Makefile` does `diff` on that and the backend-specific output file.

Reply:
Note that both `k-bin-to-text` and `diff` are skipped on zero exit code.

Reply:
We haven't needed the output file until now, because I haven't introduced bugs which we needed to display the diff for. Only on this PR (which introduced a bug) did the file ommission show up.

Reply:
This was resolved in #380 .